src/common/LoggedInProviders.tsx

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
 
 import { ActiveAccountContextProvider } from '../hooks/useActiveAccount';
+import { NotificationsManagerProvider } from '../hooks/useNotificationManager';
+import { UnreadMessagesContextProvider } from '../hooks';
```

Comment:
Let's change this to import from the most-specific file possible, to avoid a bunch of potentially-undesired imports for SDK consumers.

Reply:
Updated - and grabbed a bunch more in the other files I touched too.